.env.dev

```
@@ -0,0 +1,122 @@
+###> symfony/framework-bundle ###
+APP_ENV=dev
+APP_DEBUG=true
+APP_SECRET=abea018c70a52433e4c5e6c2b21c7c6cae2dd72c
+APP_API_KEY=958f90547ca47a539a731684ec7f9094b36bfe4a
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The APP_API_KEY key should go before the APP_DEBUG key

(UnorderedKey)